### PR TITLE
feat: added Homepage,  products page and cart page

### PR DESCRIPTION
**Pull request by Chakravarthi0**
added Homepage,  products page and cart page.

PS: I have not chosen a theme or name yet. I will update those shortly.

### CODE REVIEW

**Review by manojkumar8400 — APPROVED** [review]
looks good.

File: homepage.css [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+.hero-description > .link {
+  margin-top: 2rem;
+}
+
+/* categories */
+
+.categories-container {
+  margin-top: 5rem;
+  background-color: var(--primary-color);
+  padding: 5rem 0rem;
```
Comment (msdeshmukh009) [inline homepage.css:44]: This can be written as `padding: 5rem 0;` as 0 doesn't need the unit.

File: homepage.css [hunk 2]
```diff
@@ -0,0 +1,110 @@
+@import url("main.css");
+
+/* hero */
+
+.hero-container {
+  height: 85vh;
+  display: grid;
+  grid-template-columns: 50% 50%;
```
Comment (msdeshmukh009) [inline homepage.css]: Maybe use fr unit here as well. As I see you are using it in the below code, just to maintain consistency.

**Review by msdeshmukh009 — APPROVED** [review]
Looking good!!

**Pull request merged**